### DISCUSSION TITLE
Value of r changed to 5 after just one step

### DISCUSSION THREAD

**Discussion by Sadam1195** [coqui-ai/TTS#460] [General Q&A] [answered]
In both dev and main branch I noticed that value of r in gradual training is changed right after 1 step, is that intended or that is something left out unknowingly?

`[first_step, r, batch_size]`
`"gradual_training": [[0, 7, 64], [1, 5, 64], [50000, 3, 32], [130000, 2, 32], [290000, 1, 32]],`

**Comment by erogol** [comment, marked as answer]
intended to kick start the training smoothly like starting with a smaller learning rate.
